samples/SQL/allowed-sql-database-collation/README.md

```diff
@@ -0,0 +1,26 @@
+# Apploved SQL databasecollations
+
+This policy requires that only Azure SQL databases with the approved collation are deployed. You specify an array of approved collations.
+
+## Try on Portal
+
+[![Deploy to Azure](http://azuredeploy.net/deploybutton.png)](https://portal.azure.com/#blade/Microsoft_Azure_Policy/CreatePolicyDefinitionBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2FAzure%2Fazure-policy%2Fmaster%2Fsamples%2FSQL%2Fallowed-sql-database-collation%2Fazurepolicy.json)
+
+## Try with PowerShell
+
+````powershell
+$definition = New-AzPolicyDefinition -Name "allowed-sql-database-collation" -DisplayName "Approved Azure SQL database collations" -description "This policy requires that only Azure SQL databases with the approved collation are deployed" -Policy 'https://raw.githubusercontent.com/Azure/azure-policy/master/samples/SQL/allowed-sql-database-collation/azurepolicy.rules.json' -Parameter 'https://raw.githubusercontent.com/Azure/azure-policy/master/samples/SQL/allowed-sql-database-collation/azurepolicy.parameters.json' -Mode All
```

Comment:
Change the displayName in PS/CLI examples to match the display name at the top of the Readme